roles/nextcloud/templates/docker-compose.nextcloud.yml.j2

```
@@ -31,8 +31,8 @@ services:
 
   nextcloud:
     container_name: nextcloud
-    #image: nextcloud:{{ nextcloud.version }}
-    build: ./build
+    image: nextcloud:{{ nextcloud.version }}
+    #build: ./build
```

```suggestion
    #image: nextcloud:{{ nextcloud.version }}
    build: ./build
```

this is why this needs to be build.
https://github.com/VivumLab/VivumLab/pull/234#issue-545780644

---

About that other PR you mentioned, I have had problems with the `:fpm` image on my current instance and always more success with the `:apache` one. What if we changed to that instead and skipped the `nextcloud_web` container? The `:apache` image already contains the proxy and is capable of serving to traefik...

---

I'll try something later, this worked for me and a couple of other people without a problem. Just curios why you are having a hard time.